pages/base_page.py

```diff
@@ -15,5 +15,8 @@ def __init__(self, context):
     def navigate(self):
         self.context.driver.get(self.context.base_url+self.url)
 
+    def navigate2(self):
```

Comment:
Please make method name more informative.

Reply:
It was redundant. Forgot to remove after trying to do 2nd window browser.

Reply:
Done